index.html

```diff
@@ -42,6 +42,26 @@
                 border-left-width: 4px;
                 border-left-style: solid;
             }
+            #UNCONFIRMED {
+                border-style: solid;
+                border-width: 1;
+	        }	
+            #CONFIRMED {
+                border-style: solid;
+                border-width: 1;
+	        }	
+            #IN_PROGRESS {
+                border-style: solid;
+                border-width: 1;
+	        }	
+            #RESOLVED {
+                border-style: solid;
+                border-width: 1;
+	        }	
+            #VERIFIED {
+                border-style: solid;
+                border-width: 1;
+	        }	
```

Comment:
Ah, can you rebase onto master, I did some card styling last night that actually does the same thing so all these won't be needed anymore.